diesel/src/query_dsl/limit_dsl.rs

```diff
@@ -16,6 +16,38 @@ where
 {
     type Output = <T::Query as LimitDsl>::Output;
 
+    /// Set the `LIMIT` on the query.
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// # #[macro_use] extern crate diesel_codegen;
+    /// # #[macro_use] extern crate diesel;
+    /// # include!("../doctest_setup.rs");
+    /// # table! {
+    /// #     users {
+    /// #         id -> Integer,
+    /// #         name -> VarChar,
+    /// #     }
+    /// # }
+    ///
+    /// # #[derive(Queryable, PartialEq, Debug)]
+    /// # struct User {
+    /// #     id: i32,
+    /// #     name: String,
+    /// # }
+    ///
+    /// # fn main() {
+    /// #     let connection = establish_connection();
+    /// let user1 = NewUser { name: "Sean".into() };
```

Comment:
This could probably use an ad-hoc insert more and be more clear, couldn't it?

```rust
insert_into(users::table)
    .values(vec![
        users::name.eq("Sean"),
        users::name.eq("Tess"),
        users::name.eq("Pascal"),
    ])
```